content/_data/changelogs/weekly.yml

```diff
@@ -7719,6 +7719,134 @@
 
   # pull: 4732 (PR title: Fix hyperlink in maintainers doc)
 
+- version: "2.239"
+  date: 2020-06-01
+  banner: >
+    Jenkins 2.239 was not packaged or delivered.
+    All changes planned for 2.239 are included in 2.240.
+  changes:
+  - type: bug
+    category: internal
+    authors:
+    - kohsuke
+    message: |-
+      Jenkins 2.239 was not placed in the artifact repository or on the download site.
+
+- version: "2.240"
+  date: 2020-06-01
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4752
+    issue: 60924
+    authors:
```

Comment:
There might be some confusion here. These 3 authors contributed to the Jenkins code part indeed, but the list of contributors to the dark theme is longer. No strong opinion whether and how to reflect it there. https://github.com/jenkinsci/dark-theme/blob/master/CONTRIBUTORS.md

Reply:
We only note core contributors here, it's the core changelog, not the ecosystem changelog.

Reply:
Yeah. But dark theme is not a "core changelog" either. The internal changes made it possible, but there is no Dark Theme shipped in the Jenkins core itself. It was me who proposed to add it for the record

Reply:
I'm open to either.